<launch>
    <!-- Lost robot starts in LOST_DISTRESS -->
    <node pkg="team04" type="biorescue_phase1.py" name="biorescue_lost" output="screen">
        <param name="is_lost" value="true"/>
        <param name="distress_volume" value="200"/>
        <param name="confirm_volume" value="180"/>
        <param name="rumble_volume" value="180"/>
    </node>
</launch>
